NEWS.md

```diff
@@ -22,6 +22,12 @@
   (notably `PooledArray` and `CategoricalArray`) or when they contained only
   integers in a small range.
   ([#2812](https://github.com/JuliaData/DataFrames.jl/pull/2812))
+* the `unstack` function receives new keyword argument `fillvalue`
```

Comment:
Maybe we should call this just `fill`? Or is that name useful in other contexts as a Boolean argument?

Reply:
No - this name is specific only to this function. I will rename it to `fill`.